index.html

```diff
@@ -1151,6 +1155,40 @@ <h2>Constrainable Properties</h2>
         </tbody>
       </table>
     </section>
+    <section>
+      <h2>Capture timestamps</h2>
+      <div>
+          Some video sources can supply information about when a video frame was captured.
+          This information is useful for example for AV sync and end-to-end delay measurement.
+        </p>
+        <p>
+          {{VideoFrameMetadata}}.{{VideoFrameMetadata/captureTime}} MUST be set by the user
+          agent in video frames sourced from {{MediaDevices/getUserMedia()}},
+          {{MediaDevices/getDisplayMedia()}}, and {{HTMLMediaElement/captureStream()}}.
+        </p>
+        <section>
+          <h3>{{VideoFrameMetadata}}</h3>
+          <pre class="idl">
+    partial dictionary VideoFrameMetadata {
+      DOMHighResTimeStamp captureTime;
+    };</pre>
+          <section class="notoc">
+            <h4>Members</h4>
+            <dl class="dictionary-members" data-link-for="VideoFrameMetadata" data-dfn-for="VideoFrameMetadata">
+              <dt><dfn><code>captureTime</code></dfn> of type <span
+                class="idlMemberType">DOMHighResTimeStamp, readonly</span></dt>
+              <dd>
+                <p>
+                  The capture time of the frame, defined as {{Performance.timeOrigin}} + {{Performance.now()}}.
+                  This is the user agent's best estimate of the instant the frame content was captured or
+                  generated.
```

Comment:
What is the videoFrame.[timestamp](https://www.w3.org/TR/webcodecs/#dom-videoframe-timestamp) of these frames? Why do we need another one?

This is in milliseconds when the videoFrame.[timestamp](https://www.w3.org/TR/webcodecs/#dom-videoframe-timestamp) is in microseconds. That seems inconsistent.

Reply:
Hi, @jan-ivar - I was a little trigger happy in uploading this PR, I will upload changes soon.

Reply:
`captureTime` is absolute which enables delay measurements in a video pipeline. Also, video capture OS APIs typically provide media sampling timestamps which I've measured to be sometimes tens of milliseconds before they're available in a capture sink. I've tried to update what the use of the timestamps are in non-normative text.

I don't know why `VideoFrame.timestamp` was specified in micros. DOMHighResTimestamp with resolution around 0.1ms seems fine.